Libraries/LibWasm/AbstractMachine/Validator.cpp

```diff
@@ -2030,6 +2049,85 @@ VALIDATE_INSTRUCTION(if_)
     return {};
 }
 
+// https://webassembly.github.io/exception-handling/core/valid/instructions.html#xref-syntax-instructions-syntax-instr-control-mathsf-throw-x
+VALIDATE_INSTRUCTION(throw_)
+{
+    auto tag_index = instruction.arguments().get<TagIndex>();
+    TRY(validate(tag_index));
+
+    auto tag_type = m_context.tags[tag_index.value()];
+    auto& type = m_context.types[tag_type.type().value()];
+    if (!type.results().is_empty())
+        return Errors::invalid("throw type"sv, "empty"sv, type.results());
+
+    for (size_t i = 0; i < type.parameters().size(); ++i)
+        TRY(stack.take(type.parameters()[type.parameters().size() - i - 1]));
```

Comment:
`for (auto& parameter : type.parameters().in_reverse()) ...` ?

Reply:
ah nice, I bet I was in "spec words to text" mode writing that :P